examples/configure_with_bazel_transitive/BUILD

```diff
@@ -5,7 +5,6 @@ cc_library(
     name = "built_with_bazel",
     srcs = ["builtWithBazel.c"],
     hdrs = ["builtWithBazel.h"],
-    includes = ["."],
```

Comment:
This might be something that needs to be accounted for in this PR.

Reply:
Yes, I'm trying to figure out whether this was just a mistake here to include the entire directory, or whether we need to have a look at `cmake_external()` to find where (and why) does it do the copy.

You're right, users should be able to set `includes = ["."]`.

Reply:
Actually, I think this would not be a problem if we would not put `BUILD_TMPDIR` under `INSTALLDIR`. Which I think is a change I wanted to make anyway — I just found [this comment here](https://github.com/bazelbuild/rules_foreign_cc/pull/367/files/6fd652d2abe0a3245930b8ba7d4cdc7e993ad0c9#diff-91a54d3b1ad27c4628f09d7588304ca88968f04fabfe821eb0398904f6e5e1c0).

Reply:
I abandoned this change, instead I'm now trying to put the build dir and the deps dir outside of the install dir, which is the right thing to do anyway IMO. Let's see if CI is happy.